_posts/2020-07-24-when-scenarios-and-reality-diverge/when-scenarios-and-reality-diverge.Rmd

```diff
@@ -0,0 +1,380 @@
+---
+title: "When Scenarios and Reality Diverge"
+author:
+  - name: Vincent Jerosch-Herold
+    url: https://github.com/vintented
+date: 07-24-2020
+output: 
+  distill::distill_article:
+    toc: true
+    self_contained: true
+categories:
+  - other
+---
+
+```{r setup, include=FALSE}
+library(gt)
+library(tidyverse)
```

Comment:
Maybe pull calls to `library()` into a chunk visible in the output. If anyone tries to reproduce your code, they will want to know which packages you used.